docs/flink/flink-getting-started.md

```diff
@@ -44,14 +43,14 @@ Iceberg integration for Apache Flink is available for Flink versions 1.12, 1.13,
 
 ## Preparation when using Flink SQL Client
 
-To create iceberg table in flink, we recommend to use [Flink SQL Client](https://ci.apache.org/projects/flink/flink-docs-stable/dev/table/sqlClient.html) because it's easier for users to understand the concepts.
+To create iceberg table in flink, we recommend to use [Flink SQL Client](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/sqlclient/) because it's easier for users to understand the concepts.
```

Comment:
Why change this link to nighlies.apache.org ?  I see the ci.apache.org will redirect to the target page correctly ?

Reply:
+1 to not using nightlies.

Like Flink, we template our docs based on the Iceberg version now so it might be possible to do the same.

Otherwise, it used to be the case that the “latest” version (latest Spark version supported etc) could be automatically updated when the site was deployed.

So we might see if we can link to the correct version of the Flink docs, but otherwise the stable latest is better than nightlies imo.

Reply:
updated

Reply:
@openinx updated.